apple/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -402,7 +408,8 @@ - (void)maybeDropAncestors:(REAUIView *)child
 }
 
 - (BOOL)startAnimationsRecursive:(REAUIView *)view
-    shouldRemoveSubviewsWithoutAnimations:(BOOL)shouldRemoveSubviewsWithoutAnimations;
+    shouldRemoveSubviewsWithoutAnimations:(BOOL)shouldRemoveSubviewsWithoutAnimations
+                            shouldAnimate:(BOOL)shouldAnimate;
```

Comment:
Now when I think about it, `shouldAnimate` is set only by the skipper wrapper. How about we rename `shouldAnimate` to something more specific to the usage of that wrapper like `isExitingEnabled`?